compiler/rustc_next_trait_solver/src/solve/inspect/build.rs

```diff
@@ -34,9 +24,22 @@ use rustc_type_ir::{self as ty, InferCtxtLike, Interner};
 /// trees. At the end of trait solving `ProofTreeBuilder::finalize`
 /// is called to recursively convert the whole structure to a
 /// finished proof tree.
-pub(in crate::solve) struct ProofTreeBuilder<Infcx, I = <Infcx as InferCtxtLike>::Interner>
+use std::marker::PhantomData;
```

Comment:
why did you move the imports below the doc comment?

Reply:
Accident, I think. I deleted all of the imports in every file and probably just accidentally added it here.